### PR TITLE
Updates ds global script & campaign run title field 

### LINKED ISSUE #6016: Translate title field on campain run node
Use the title module to translate the title of campaign run nodes

refs #5987

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?
- Makes the campaign run title field translatable by using title module
- Updates ds global script to update existing campaign run titles
#### How should this be manually tested?

```
dosomething@dev:/var/www/dev.dosomething.org/html$ drush ds-global-titles-fix campaign_run
Replacing field values for  in campaign_run node.
Done.                                                                                                                                                                  [ok]
dosomething@dev:/var/www/dev.dosomething.org/html$ 
```
#### What are the relevant tickets?

Fixes #6016 
#### Screenshots (if appropriate)

screen shot 2016-01-13 at 11 15 02 am

### CODE REVIEW

**Comment by itsjoekent** [comment]
oh and heads up @angaither @sbsmith86 ! :siren:

File: bin/ds-global [hunk 1]
```diff
@@ -61,6 +61,7 @@ function enable {
 
   # Fix Campaign translatable title values.
   drush ds-global-titles-fix campaign
+  drush ds-global-titles-fix campaign_run
```
Comment (sergiitk) [inline bin/ds-global]: No, that's not the right place for fixing title. Let me find you an example.

Comment (sergiitk) [inline bin/ds-global]: Please take a look at #5892.

Comment (itsjoekent) [inline bin/ds-global]: so stick that drush addition I made in a new function like `make_more_campaign_fields_translatable`, and then add it to the else if block below so you can specify it

Comment (sergiitk) [inline bin/ds-global]: Sounds good

**Comment by itsjoekent** [comment]
@sergii-tkachenko ready for re-review!

**Comment by sergiitk** [comment]
:+1:

**Comment by sergiitk** [comment]
Make sure to run `ds-global enable-campaign-runs` after deploying!

**Pull request merged**